Documentation/Signoffs/MasterMCU.md

```diff
@@ -0,0 +1,41 @@
+MASTER MICROCONTROLLER
+------------------------
+### Function of the Subsystem
+The master microcontroller (MMCU) must receive data from four seperate local microcontrollers that interface with the
+sensors in other subsystems. This MMCU will read serial data from each local MCU and then send that data to the PC to
+be processed in the simulation.
+
+### Specifications & Constraints
+- Must connect to no less than four local microcontrollers
+- Must provide real-time data transfer
+- Shall connect to the PC running the simulation
+- Shall have sufficient GPIO pins for communication protocols
+- Shall be powered by the PC
+
+### Wiring Schematic
+
+![image](https://user-images.githubusercontent.com/100802413/202863978-89cc0d7a-06cd-4679-a6c5-b71eb77ff12f.png)
+
+###### _Figure 1: Master & Slave Connections_
+
+### Analysis
+##### 1. Microcontroller Selection
+The MCU will be the Arduino Mega 2560 that includes a 16 MHz clock frequency, 54 digtial I/O pins, 4 UARTs, and can be powered via USB. This MCU supports serial communication with up to 4 other local microcontrollers via UART which meets the constraint above. Additionally, the board can be programmed using the Arduino IDE which includes multiple libraries and functions to help manage multiple communications via UART simultaneously without interfering with the data transmitted or received. The Arduino Mega also includes I2C pins for an alternate form of communication.
+##### 2. Communication Protocols
```

Comment:
You haven't shown that the microcontroller can actually do Uart to all the other controllers.